dotcom-rendering/scripts/webpack/webpack.config.browser.js

```diff
@@ -62,44 +62,10 @@ module.exports = ({ isLegacyJS, sessionId }) => ({
 			{
 				test: /\.[jt]sx?|mjs$/,
 				exclude: module.exports.babelExclude,
-
-				use: [
-					{
-						loader: 'babel-loader',
-						options: {
-							presets: [
-								'@babel/preset-react',
-								isLegacyJS
-									? [
-											'@babel/preset-env',
-											{
-												targets: {
-													ie: '11',
-												},
-												modules: false,
-											},
-									  ]
-									: [
-											'@babel/preset-env',
-											{
-												bugfixes: true,
-												targets: {
-													esmodules: true,
-												},
-											},
-									  ],
-							],
-							compact: true,
-						},
-					},
-					{
-						loader: 'ts-loader',
-						options: {
-							configFile: 'tsconfig.build.json',
-							transpileOnly: true,
-						},
-					},
-				],
+				use: {
+					loader: 'swc-loader',
+					options: swcConfig,
```

Comment:
It would be nice to have a typed config, if at all possible! Some packages have a `defineConfig` method for this purpose.